students/brian_warn/session05/except_exercise.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+
+"""
+An exercise in playing with Exceptions.
+Make lots of try/except blocks for fun and profit.
+
+Make sure to catch specifically the error you find, rather than all errors.
+"""
+
+from except_test import fun, more_fun, last_fun
+
+
+# Figure out what the exception is, catch it and while still
+# in that catch block, try again with the second item in the list
+first_try = ['spam', 'cheese', 'mr death']
+
+try:
+    joke = fun(first_try[0])
+except NameError as ne:
+    print("fun() is not coded properly: " + str(ne))
+
```

Comment:
I like what you did here, showing the actual exception that was raised is a great thing to learn. I may incorporate this in the next iteration of this exercise. However, I did also want you try again (in the except block) to call the function, this time with the second item in the first_try list. No biggie, you just missed some spam. ;-)